concent_api/concent_api/settings/base.py

```diff
@@ -294,6 +294,9 @@
 # rendering results and any intermediate files.
 # VERIFIER_STORAGE_PATH = ''
 
+# A Global constant defining chunk_size used for downloading files from nginx storage to verifier_storage.
+VERIFIER_DOWNLOAD_CHUNK_SIZE = 1000000
```

Comment:
since it's defined here, is there any point of defining it in `development` and `testing` (especially when it has the same value)?

Reply:
After a quick talk with @cameel he said that if setting is in `base.py` it shouldn't be in `development.py` or `testing.py`

Reply:
Yeah. @dybi is right. We add stuff to `development.py` and `testing.py` only when either there's no default in `base.py` but we want one in development/unit tests or when we want a different default.

In this case default is defined and should be usable both in production and in development so there's no need to put it in those other files.